index.js

```diff
@@ -16,16 +16,15 @@ let colors = {
     brown: '#964B00'
 };
 
-// Not sure if this is an acceptable way to do this.  Was trying to cut down on lines of code.
-const redSwatch = document.getElementById('red').style.backgroundColor = colors.red
-const blueSwatch = document.getElementById('blue').style.backgroundColor = colors.blue
-const greenSwatch = document.getElementById('green').style.backgroundColor = colors.green
-const yellowSwatch = document.getElementById('yellow').style.backgroundColor = colors.yellow
-const whiteSwatch = document.getElementById('white').style.backgroundColor = colors.white
-const blackSwatch = document.getElementById('black').style.backgroundColor = colors.black
-const orangeSwatch = document.getElementById('orange').style.backgroundColor = colors.orange
-const purpleSwatch = document.getElementById('purple').style.backgroundColor = colors.purple
-const brownSwatch = document.getElementById('brown').style.backgroundColor = colors.brown
+const swatches = {}
+
+function initializeSwatches() {
+    for (const [color, hex] of Object.entries(SUPPORTED_COLORS)) {
+        console.log(color, hex)
+        swatches[color] = document.getElementById(color)
+        swatches[color].style.backgroundColor = hex   
+    }    
+}
```

Comment:
I need to put this in a function now, because I need to defer it's invocation until after the span tags have been added [here](https://github.com/StowCode/PixelWarVanilla/pull/1/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R105)